Content.Client/_RMC14/Dropship/Utility/DropshipUtilityPointVisualizerSystem.cs

```diff
@@ -0,0 +1,64 @@
+using Content.Shared._RMC14.Dropship.AttachmentPoint;
+using Content.Shared._RMC14.Dropship.Utility;
+using Robust.Client.GameObjects;
+using Robust.Shared.Utility;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Client._RMC14.Dropship.Utility;
+
+public sealed partial class DropshipUtilityPointVisualizerSystem : VisualizerSystem<DropshipUtilityPointComponent>
+{
+    [Dependency] private readonly AppearanceSystem _appearance = default!;
+    protected override void OnAppearanceChange(EntityUid uid, DropshipUtilityPointComponent component, ref AppearanceChangeEvent args)
```

Comment:
You should be able to replace this system with GenericVisualizer in YAML, we have some examples of it in RMC14 YAMl and code